pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -156,13 +156,17 @@ filterAndCreateOverrides {
     {
       cudaAtLeast,
       gmp,
+      expat,
+      stdenv,
       lib,
     }:
     prevAttrs: {
       buildInputs =
         prevAttrs.buildInputs
         # x86_64 only needs gmp from 12.0 and on
-        ++ lib.lists.optionals (cudaAtLeast "12.0") [ gmp ];
+        ++ lib.lists.optionals (cudaAtLeast "12.0") [ gmp ]
+        # aarch64,sbsa needs expat
+        ++ lib.lists.optionals (stdenv.hostPlatform.isAarch64) [ expat ];
```

Comment:
Dear reviewers and maintainers, please review and edit this part as needed.

I'm not quite sure about using `stdenv` in here. Probably there could be better implementation here.

Reply:
Might as well ask for `hostPlatform` from `callPackage` but this looks good as is, thanks

Reply:
@SomeoneSerge 

Friendly reminder and gentle nudge here:
- Could this be merged? (I don't have permission, just in case)
- You may want to add `6.topic: cuda` label on this PR and [issue#327222](https://github.com/NixOS/nixpkgs/issues/327222)

Reply:
Thank you and sorry about the delay